docs/browseV2/browse-paths-v2.md

```diff
@@ -0,0 +1,47 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# Generating Browse V2 Aspects
```

Comment:
Let's make the name less technical:

"Generating Browse Paths (V2)"

Reply:
Let's have top-level sections underneath like:

Introduction

<What is a browse path, why does it exist for users, what is the structure of it> 

Generating Custom Browse Paths

Additional Resources

Reply:
definitely!